vendor/react-window/dist/index.esm.js

```diff
@@ -1037,32 +1139,42 @@ function createListComponent(_ref) {
               return null;
             }
 
-            var direction = _this.props.direction; // HACK According to the spec, scrollLeft should be negative for RTL aligned elements.
-            // Chrome does not seem to adhere; its scrolLeft values are positive (measured relative to the left).
-            // See https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollLeft
-
+            var direction = _this.props.direction;
             var scrollOffset = scrollLeft;
 
             if (direction === 'rtl') {
-              if (scrollLeft <= 0) {
-                scrollOffset = -scrollOffset;
-              } else {
-                scrollOffset = scrollWidth - clientWidth - scrollLeft;
+              // TRICKY According to the spec, scrollLeft should be negative for RTL aligned elements.
+              // This is not the case for all browsers though (e.g. Chrome reports values as positive, measured relative to the left).
+              // It's also easier for this component if we convert offsets to the same format as they would be in for ltr.
+              // So the simplest solution is to determine which browser behavior we're dealing with, and convert based on it.
+              switch (getRTLOffsetType()) {
+                case 'negative':
+                  scrollOffset = -scrollLeft;
+                  break;
+
+                case 'positive-descending':
+                  scrollOffset = scrollWidth - clientWidth - scrollLeft;
+                  break;
               }
-            }
+            } // Prevent Safari's elastic scrolling from causing visual shaking when scrolling past bounds.
 
+
+            scrollOffset = Math.max(0, Math.min(scrollOffset, scrollWidth - clientWidth));
             return {
               isScrolling: true,
               scrollDirection: prevState.scrollOffset < scrollLeft ? 'forward' : 'backward',
               scrollOffset: scrollOffset,
               scrollUpdateWasRequested: false
             };
           }, _this._resetIsScrollingDebounced);
-        });
-      };
+        };
+      });
```

Comment:
Looks like this and https://github.com/bvaughn/react-devtools-experimental/pull/352/files#diff-a55231897cc724978f28b3edafc87568R1199 are the causes for the error I was seeing in https://github.com/bvaughn/react-window/issues/299

Reply:
Yeah, this looks like a bad merge on my part. Thanks for spotting it.

I'm curious though- When you say that you're "seeing an error" - where are you seeing it?

Reply:
I ask because I realize now that my local project has not been using this vendored copy like I thought it was and I'm not sure when that broke :| I'll fix both...

Reply:
Fixed in 1178d57. Thanks for pointing this out.